fixit/common/config.py

```diff
@@ -43,6 +43,10 @@
     + r": (?P<codes>([-_a-zA-Z0-9]+,\s*)*[-_a-zA-Z0-9]+)"
     + r"(:( (?P<reason>.+)?)?)?"
 )
+HAS_LINT_IGNORE_REGEXP: Pattern[bytes] = re.compile(rb"# lint-(ignore|fixme):")
+HAS_LINT_IGNORE_OR_NOQA_REGEXP: Pattern[bytes] = re.compile(
+    rb"# (lint-(ignore|fixme):|noqa|flake8)"
+)
```

Comment:
Alternative to use the other patterns already defined instead of defining new ones.
 
```suggestion
HAS_LINT_IGNORE_REGEXP: Pattern[bytes] = config.LINT_IGNORE_REGEXP
HAS_LINT_IGNORE_OR_NOQA_REGEXP: Pattern[bytes] = re.compile(
    "|".join([config.LINT_IGNORE_REGEXP, config.NOQA_INLINE_REGEXP, config.NOQA_FILE_RULE, config.FLAKE8_NOQA_FILE])
)
```

Reply:
Good idea!

Reply:
The suggestion code has an undefined name `config` and requires a fix.